documentation/docs/admin/roles/access-control/labels.md

```diff
@@ -1,41 +1,55 @@
 # Labels for access control
 
 
-Label-based access control in PMM allows you to manage who has access to metrics based on labels. By creating roles, you can specify which data can be queried based on specific label criteria, for instance, allowing the QA team to view data related to test environments. 
+Label-based access control in PMM allows you to manage who has access to metrics based on labels. By creating roles, you can specify which data can be queried based on a specific criterion, for instance, allowing the QA team to view data related to test environments. 
  
-With Label-based access control, you can associate multiple labels with a role, ensuring only data from series that match your defined labels is returned. 
+With label-based access control, you can associate multiple labels with a role, ensuring only data from series that match your defined labels is returned. 
 
 
 ## Standard vs custom labels
 
-PMM supports standard as well as custom labels. PMM automatically assigns standard labels. You can also set standard labels when an object (Node, Service, or Agent) is created. Custom labels are assigned and updated only by a user.
+PMM supports standard as well as custom labels. When the user adds a service to monitoring, PMM automatically assigns some standard labels. Thease labels are mostly based on the type of service being added to monitoring, such as `service_type`, `agent_type`, `node_name`, etc. Some other labels are auto-generated by PMM, for example, `service_id`, `node_id`, etc. Normally, the user can override standard labels when an object &mdash; Node, Service, or Agent &mdash; is created. There is also a possibility to define custom labels when adding objects. Custom labels, however, are assigned and updated only by the user.
 
-**Examples**
+It's important to note that standard labels are not always available for all objects. For example, the `service_type` label is only available for services, while the `agent_type` label is only available for agents.
```

Comment:
metrics collected by agents related to services inherit their labels.

Reply:
I made an edit, please review again. 

I have a feeling that this particular note about label propagation to metrics should be moved elsewhere, to the client docs, for instance. :)